File: POC/MauiClientPOC/MauiClientPOC/RestAPIClient/Contracts/LoginModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace RestAPIClient.Contracts
{
    public class LoginModel
    {
        public Session Session { get; set; }
        public string userName { get; set; }
        public string password { get; set; }
        public bool SessionIsValidRequest { get; set; }
    }
}

File: 4WinsGame/4WinGame/AngularWebFrontend/4WinGame/src/app/Models/LoginModel.ts
import { SessionModel } from "./SessionModel";

export class LoginModel
{
    constructor(userName?:string, password?:string, session?:SessionModel, sessionIsValidRequest?:boolean){
        this.password = password;
        this.userName = userName;
        this.SessionIsValidRequest = sessionIsValidRequest;
        this.Session = session;
    }
    userName: string | undefined;
    password: string | undefined;
    Session: SessionModel | undefined;
    SessionIsValidRequest: boolean | undefined;
}